edgelet/edgelet-http/src/lib.rs

```diff
@@ -424,3 +435,27 @@ impl HyperExt for Http {
         })
     }
 }
+
+#[allow(dead_code)]
```

Comment:
Why not `cfg(unix)` it instead of `allow(dead_code)` ?

Reply:
Ah, it *is* used as the parameter regardless. In that case I suggest `#[cfg_attr(not(unix), allow(dead_code))]`

Reply:
Looks better. Thanks!